### DISCUSSION TITLE
Ansible

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1575] [Q&A] [answered]
![Screenshot 2023-11-07 171802](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/65ba52b1-b973-4193-b3d3-e29f5323739f)
It shows error

**Comment by sreedharm07** [comment, marked as answer]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/0a19e444-0acc-4726-a83d-63272e5ae715)
1 st - after hosts: there is no space 
2 nd - its supposed to be tasks
3 rd -  between - and name there should be space ( - name )
4 th - last line is not following indentation
